newtmgr/cli/res.go

```diff
@@ -259,12 +240,7 @@ func resDeleteCmd(cmd *cobra.Command, args []string) {
 		nmUsage(nil, err)
 	}
 
-	rt, err := sesn.ParseResType(args[0])
-	if err != nil {
-		nmUsage(cmd, err)
-	}
-
-	path := args[1]
+	path := args[0]
 
 	var m map[string]interface{}
 	m, err = extractResKv(args[2:])
```

Comment:
After the change, should this say args[1:] here?

Reply:
Previously, type was in args[0], so with the removal of type, path is in args[0]. Since the other args would include key-value pairs, shouldn't they be excluded from the path? Also, path is passed as a string, and won't accept a string slice in assignment. I got the following errors when making the change: 

cli/res.go:198:9: cannot use path (type []string) as type string in assignment
cli/res.go:243:9: cannot use path (type []string) as type string in assignment

Reply:
You're not starting key/value pair parsing from the argument following path. You're skipping the one right after it. On this line you need to say:
m, err = extractResKv(args[1:])

Reply:
Ah sorry, I misunderstood. I made the the appropriate changes.